src/libraries/System.Net.Sockets/tests/FunctionalTests/SendFile.cs

```diff
@@ -104,6 +105,94 @@ public void NotConnected_ThrowsException()
             }
         }
 
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public async Task SendFile_Empty_SucceedsSendingNothing(bool useAsync)
```

Comment:
Should we have a test that sends pre/post buffers with empty file name?

Reply:
Sure, I can add that.